db/re/item_db.conf

```diff
@@ -61939,6 +61940,11 @@ item_db: (
 	Slots: 1
 	Loc: 256
 	View: 525
+	Script: <"
+		bonus bDex, 2;
+		skill TF_DOUBLE, 5;
```

Comment:
Does it mean, if a thief has level 10 Double Attack  equipped the item which give him another Level 5 Double Attack, hence result in a Level 15 Double Attack ?